docs/usage.rst

```diff
@@ -296,6 +333,35 @@ use the **verify-files** command:
       -> expected checksum adler32:62e0c299, found adler32:62e0c299
     ==> Success!
 
+We can verify files just after downloading files with help of **download-files --verify** command.
```

Comment:
If we switch to verifying each file after it is downloaded (see above), then we can also amend this example.

(We could download all 4 files and then verify all 4 files, but if there are some connection troubles etc, it seems safer to download first file and verify it, then proceed to downloading second file, etc.)